Antidote/CallsManager.m

```diff
@@ -0,0 +1,331 @@
+//
+//  CallsManager.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/13/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "CallsManager.h"
+#import "ProfileManager.h"
+#import "OCTSubmanagerCalls.h"
+#import "DialingCallViewController.h"
+#import "RingingCallViewController.h"
+#import "ActiveCallViewController.h"
+#import "AppDelegate+Utilities.h"
+#import "RBQFetchedResultsController.h"
+#import "ProfileManager.h"
+#import "Helper.h"
+#import "OCTCall.h"
+#import "TabBarViewController.h"
+#import "AbstractCallViewController.h"
+
+@interface CallsManager () <RBQFetchedResultsControllerDelegate, ActiveCallViewControllerDelegate, DialingCallViewControllerDelegate, RingingCallViewControllerDelegate>
```

Comment:
It would be nice to have some detailed logging for CallsManager.

It is not part of this pull request, just to let you know for the future.